torch/testing/_internal/common_methods_invocations.py

```diff
@@ -1253,8 +1270,6 @@ def method_tests():
         ('complex', (S, S, S), ((S, S, S),), ''),
         ('abs', (S, S, S), NO_ARGS, '', (True,)),
         ('abs', (), NO_ARGS, 'scalar', (True,)),
```

Comment:
Reviewer's note:

There is another test for angle but it also tests abs:

https://github.com/pytorch/pytorch/blob/65876d3f5134bba3c6ab5dccd1860ce1b9e43b6d/test/test_unary_ufuncs.py#L538